packages/core/src/modules/ledger/services/IndyLedgerService.ts

```diff
@@ -131,6 +140,9 @@ export class IndyLedgerService {
     }
   }
 
+  /**
+   * @deprecated
+   */
   public async getEndpointsForDid(agentContext: AgentContext, did: string) {
```

Comment:
I see you moved some methods to the indy pool service. Can we update all methods in the indy ledger service to call those new methods instead? As they're private in this class we can migrate them (instead of duplicate) without any issues